docs/user_guide/security/auditing.rst

```diff
@@ -5,10 +5,16 @@ Auditing
 NVFLARE has an auditing mechanism to record events that occur in the system. Both user command events
 and critical job events generated by the learning process are recorded.
 
+The audit logging system is primarily implemented in :class:`Auditor<nvflare.fuel.sec.audit.Auditor>`.
+
 Audit File Location
 ^^^^^^^^^^^^^^^^^^^^
 The audit file audit.txt is located in the root directory of the workspace.
 
+Note that job related log messages are in the Server Job (SJ)'s audit log file, not in the Server Parent (SP)'s audit file.
+
+To see SJ's audit file, you need to run "download_job <job_id>" after the job is done, and you will see the audit log file in the "workspace" folder of the downloaded job.
+
```

Comment:
Can you add similar sentences for CP/CJ?

Reply:
Yes, I will add that